voxblox/include/voxblox/utils/protobuf_utils.h

```diff
@@ -10,13 +11,14 @@
 namespace voxblox {
 
 namespace utils {
-bool readProtoMsgCountToStream(std::fstream* stream_in, uint32_t* message_count,
-                               uint64_t* byte_offset);
+bool readProtoMsgCountFromStream(std::istream* stream_in,
```

Comment:
This is naming correction that will break depending packages if they've implemented their own loading functions. 

Obviously this function is reading _from_ a stream, rather than _to_ a stream.

Reply:
I think you should be able to rename this. AFAIK this should only be used deep in the internals of this stuff (note to self for future designs: hide functions like these :) )

Reply:
People implementing new layers, submap types etc have to call this function, which does seem a bit odd to me, as we've discussed before :). Imma merge and let's see if anyone complains.